recipes/arm-gnu-toolchain/all/test_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualBuildEnv"
+
+    def requirements(self):
```

Comment:
This needs to be ``build_requirements(self)`` method
```suggestion
    def build_requirements(self):
```